### DISCUSSION TITLE
Support enterKeyHint for Mobile device

### DISCUSSION THREAD

**Discussion by clemdesign** [sibiraj-s/ngx-editor#365] [General]
Hello,

I would like to propose the support of [enterKeyHint](https://html.spec.whatwg.org/multipage/interaction.html#input-modalities:-the-enterkeyhint-attribute) attribute for mobile device. 

There is an example of implementation [here](https://ionicframework.com/blog/keyboard-improvements-for-ionic-apps/) (Look for `enterkeyhint`).

### Problem:

On iOS devices, the keyboard use "Back" label for "Enter" key. It is not appropriate for editing. And iOS seems not propose the "Enter" label for the keyboard (after many tests and I'm not found spec on this).

So, I propose to add an option `enterKeyHint` to `Editor` of this library, which set the attribute `enterkeyhint` of the ProseMirror editor.  
This option will allow to define all possibles values from [spec](https://html.spec.whatwg.org/multipage/interaction.html#input-modalities:-the-enterkeyhint-attribute).  
And Integrator can choose the right label according context.

I prepared an PR and can propose it.

Thanks.

**Comment by sibiraj-s** [comment]
> I prepared an PR and can propose it.

Sure.

**Comment by clemdesign** [comment]
Done: https://github.com/sibiraj-s/ngx-editor/pull/366

[a comment by an automated account is not shown]
